app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,93 @@
+require 'chronic'
+
+class TasksController < ApplicationController
+  def index #homepage
+    @all_tasks = Task.all.order(date_completed: :desc)
+  end
+
+  def new
+    @task = Task.new
+    @people = Person.all
+    # @task = Task.find(params[:id])
+
+    render "new"
+  end
+
+  def create
+    @task = Task.new(tasks_create_params[:task])
+    # @task.due_date == nil ? "" : @task.due_date = Time.parse(params[:due_date])
+    # @task.due_date = Chronic.parse(params[:due_date])
+    # defaults = {:status => "Not Started"}
+    # params = defaults.merge(params)
+    # @task = params[:status]
+    person = Person.find(params[:people])
+    person.tasks << @task
+
+    if @task.save
+      redirect_to root_path
+    else
+      redirect_to action: "add"
+    end
```

Comment:
This `if/else` statement would be unnecessary if you used `.create`

Where is the `"add"` action?